### DISCUSSION TITLE
GameMode Conquest Large

### DISCUSSION THREAD

**Discussion by andrejobim** [Kiiyya/BattleFox#42] [General]
I'm trying to configure GameMode Conquest Large but I couldn't find any references within the project.

Is it only available for rush?

**Comment by Kiiyya** [comment]
Hi. Yeah the project was initially created for Rush, and we simply never tried it for Conquest. I will no longer be maintaining this project and I won't add the functionality myself, because I simply don't play BF4 anymore, but I can answer some questions if you want to add that functionality yourself. It probably shouldn't be too difficult, the first step would be to add conquest large to the `GameMode` enum and then see what breaks, or maybe it'll just work, I have no idea honestly :D.
